include/erlcloud_ddb2.hrl

```diff
@@ -388,7 +388,7 @@
         {global_secondary_indexes :: undefined | [#ddb2_global_secondary_index_info{}],
          local_secondary_indexes  :: undefined | [#ddb2_local_secondary_index_info{}],
          sse_description :: undefined | erlcloud_ddb2:sse_description(),
-         stream_description :: undefined | #ddb2_stream_description{},
+         stream_description :: undefined | #ddb2_stream_description{} | {boolean(), erlcloud_ddb2:stream_view_type()},
```

Comment:
Does this seem OK? I adapted it for tests to dialyze properly, but am having seconds thoughts.

Reply:
I think this is an acceptable input, but I don't think it's correct output.

Reply:
(probably 1 more lazy meck, and that one may actually be _my_ fault).

Reply:
I'm going to take a closer look at this.

Reply:
I'll change the output (it'll be a breaking change, in any case, because the interface will change) to respect "I don't think it's correct output". Once you check it, we can probably revert it if you don't like the change. e08033d

Reply:
Answered [here](https://github.com/erlcloud/erlcloud/pull/650#discussion_r452184237), sorry about that.